src/main/kotlin/com/hypto/iam/server/service/UsersService.kt

```diff
@@ -159,12 +159,15 @@ class UsersServiceImpl : KoinComponent, UsersService {
     ) {
         val org = organizationRepo.findById(organizationId)
             ?: throw EntityNotFoundException("Invalid organization id")
+        val encodedEmail = email?.let {
+            getEmail(organizationId, subOrganizationName, email)
+        } ?: throw BadRequestException("Email is required")
```

Comment:
```suggestion
        val encodedEmail = getEmail(organizationId, subOrganizationName, email) ?: throw BadRequestException("Email is required")
```

Reply:
The problem here is the parameter "email" is non nullable in getEmail method, so this nullable check is required.